test/unit/ngsiv2/amqpBinding-test.js

```diff
@@ -0,0 +1,320 @@
+                done();
+            }, 100);
+        });
+    });
+
+    describe('When a message with multiple measure groups arrives to a Device routing key', function() {
+        beforeEach(function() {
+            contextBrokerMock
+                .matchHeader('fiware-service', 'smartGondor')
+                .matchHeader('fiware-servicepath', '/gardens')
+                .post('/v2/entities/Second%20UL%20Device/attrs',
+                    utils.readExampleFile('./test/unit/ngsiv2/contextRequests/singleMeasure.json'))
+                .reply(204);
+
+            contextBrokerMock
+                .matchHeader('fiware-service', 'smartGondor')
+                .matchHeader('fiware-servicepath', '/gardens')
+                .post('/v2/entities/Second%20UL%20Device/attrs',
+                    utils.readExampleFile('./test/unit/ngsiv2/contextRequests/secondSingleMeasure.json'))
+                .reply(204);
+        });
+
+        it('should send a two update context requests to the Context Broker one with each attribute', function(done) {
+            channel.publish(config.amqp.exchange, '.1234.MQTT_2.attrs', new Buffer('temperature|23#humidity|98'));
+
+            setTimeout(function() {
+                contextBrokerMock.done();
+                done();
+            }, 100);
+        });
+    });
+    describe('When multiple groups of measures arrive, with multiple attributes, to a Device routing key', function() {
+        beforeEach(function() {
+            contextBrokerMock
+                .matchHeader('fiware-service', 'smartGondor')
+                .matchHeader('fiware-servicepath', '/gardens')
+                .post('/v2/entities/Second%20UL%20Device/attrs',
+                    utils.readExampleFile('./test/unit/ngsiv2/contextRequests/multipleMeasure.json'))
+                .reply(204);
+
+            contextBrokerMock
+                .matchHeader('fiware-service', 'smartGondor')
+                .matchHeader('fiware-servicepath', '/gardens')
+                .post('/v2/entities/Second%20UL%20Device/attrs',
+                    utils.readExampleFile('./test/unit/ngsiv2/contextRequests/secondMultipleMeasure.json'))
+                .reply(204);
+        });
+
+        it('should send a two update context requests to the Context Broker one with each attribute', function(done) {
+            channel.publish(config.amqp.exchange, '.1234.MQTT_2.attrs',
+                new Buffer('temperature|23|humidity|98#temperature|16|humidity|34'));
+
+            setTimeout(function() {
+                contextBrokerMock.done();
+                done();
+            }, 100);
+        });
+    });
+
+    describe('When a measure with a timestamp arrives with an alias to TimeInstant', function() {
+        var provisionProduction = {
+            url: 'http://localhost:' + config.iota.server.port + '/iot/devices',
+            method: 'POST',
+            json: utils.readExampleFile('./test/deviceProvisioning/provisionTimeInstant.json'),
```

Comment:
Maybe you mean ./test**/unit/ngsiv2/**deviceProvisioning/provisionTimeInstant.json. Although in fact both files seems to be exact copies...